src/benchmarks/micro/libraries/System.Runtime/Perf.String.cs

```diff
@@ -154,13 +154,20 @@ public string TrimEnd_CharArr(string s, char[] c)
             => s.TrimEnd(c);
 
         [Benchmark]
-        [Arguments("Hello", 'l', '!')] // Contains two 'l'
-        [Arguments("Hello", 'a', 'b')] // Contains one 'a'
-        [Arguments("This is a very nice sentence", 'z', 'y')] // 'z' does not exist in the string
-        [Arguments("This is a very nice sentence", 'i', 'I')] // 'i' occuress 3 times in the string
+        [ArgumentsSource(nameof(ReplaceArguments))]
         public string Replace_Char(string text, char oldChar, char newChar)
             => text.Replace(oldChar, newChar);
 
+        public static IEnumerable<object[]> ReplaceArguments()
```

Comment:
Verifyed that the method names are displayed the same way, so that the reporting tool has correlation. Is this enough to be on the safe side here?

```
|          Method |                 text | oldChar | newChar |       Mean |      Error |    StdDev |
|---------------- |--------------------- |-------- |-------- |-----------:|-----------:|----------:|
| Replace_CharNew |  ----(...)---- [100] |       - |       + |  32.553 ns |  21.554 ns | 1.1815 ns |
| Replace_CharNew | ----(...)---- [1000] |       - |       + | 202.048 ns | 178.291 ns | 9.7727 ns |
|    Replace_Char |                Hello |       a |       b |   5.256 ns |   4.023 ns | 0.2205 ns |
| Replace_CharNew |                Hello |       a |       b |   5.367 ns |   2.158 ns | 0.1183 ns |
|    Replace_Char |                Hello |       l |       ! |  17.491 ns |   4.643 ns | 0.2545 ns |
| Replace_CharNew |                Hello |       l |       ! |  18.092 ns |  12.480 ns | 0.6841 ns |
|    Replace_Char | This (...)tence [28] |       i |       I |  36.695 ns |  91.723 ns | 5.0277 ns |
| Replace_CharNew | This (...)tence [28] |       i |       I |  30.385 ns |  25.207 ns | 1.3817 ns |
|    Replace_Char | This (...)tence [28] |       z |       y |  11.598 ns |   6.366 ns | 0.3489 ns |
| Replace_CharNew | This (...)tence [28] |       z |       y |  12.132 ns |  10.304 ns | 0.5648 ns |
```
The ones with suffix _New_ are the methods with the ArgumentsSource, the other ones with Arguments only (as in main-branch).

Reply:
Sorry, I thought I had already responded here. As long as the test name or the arguments have not changed values we should have no issues here. And from what I can tell nothing has changed for the existing test cases. So this LGTM.